app/models/manageiq/providers/datawarehouse_manager.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers
+  class DatawarehouseManager < BaseManager
+    class << model_name
+      def route_key
+        "ems_datawarehouse"
```

Comment:
@martinpovolny is this a problem, that route key and singular_route_key are the same?

I see that `MiddlewareManager` has that too.